Dockerfile

```diff
@@ -1 +1,185 @@
+#
+# This one houses the main git clone
+#
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest as src
 
+#
+# Basic Parameters
+#
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="0.40.0"
+ARG PKG="google_cadvisor"
+ARG SRC="https://github.com/google/cadvisor.git"
+
+#
+# Some important labels
+#
+LABEL ORG="Armedia LLC"
+LABEL MAINTAINER="Armedia Devops Team <devops@armedia.com>"
+LABEL APP="Google cAdvisor"
+LABEL VERSION="${VER}"
+
+WORKDIR /src
+
+#
+# Download the primary artifact
+#
+RUN yum -y update && yum -y install git && git clone -b "v${VER}" --single-branch "${SRC}" "/src"
+
+#
+# This one builds the artifacts
+#
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest as build
+
+#
+# Basic Parameters
+#
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="0.40.0"
+ARG PKG="google_cadvisor"
+ARG PFM_MAJ="4"
+ARG PFM_MIN="11"
+ARG PFM_REL="0"
+ARG PFM_VER="${PFM_MAJ}.${PFM_MIN}.${PFM_REL}"
+ARG PFM_TAR="libpfm-${PFM_VER}.tar.gz"
+ARG PFM_SRC="https://sourceforge.net/projects/perfmon2/files/libpfm${PFM_MAJ}/${PFM_TAR}"
+ARG PFM_SHA="112bced9a67d565ff0ce6c2bb90452516d1183e5"
+ARG GO_VER="1.16.7"
+ARG GO_SRC="https://golang.org/dl/go${GO_VER}.${OS}-${ARCH}.tar.gz"
+
+#
+# Set the Go environment
+#
+ENV GOROOT="/usr/local/go"
+ENV GOPATH="/go"
+ENV PATH="${PATH}:${GOROOT}/bin"
+#ENV GO_FLAGS="-tags=libpfm,netgo,libipmctl"
+ENV GO_FLAGS="-tags=libpfm,netgo"
+
+#
+# Download and install go
+#
+RUN curl -L "${GO_SRC}" -o - | tar -C "/usr/local" -xzf -
+
+#
+# Some important labels
+#
+LABEL ORG="Armedia LLC"
+LABEL MAINTAINER="Armedia Devops Team <devops@armedia.com>"
+LABEL APP="Grafana"
```

Comment:
Grafana?

Reply:
I thought labels didn't matter in intermediates :D

J/k.... I fixed it :)